cloud-service-providers/google-cloud/vertexai/python/airgap/docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+services:
+  nim-llama-8b:
+    image: nvcr.io/nim/meta/llama3-8b-instruct:1.0.0
+    shm_size: 35gb
+    runtime: nvidia
+    hostname: nim-llama-8b
+    healthcheck:
+      test: python /healthcheck.py
+      interval: 1m
+      timeout: 30s
+      retries: 15
+      start_period: 30s
+    environment:
+      - NGC_API_KEY=${NGC_API_KEY}
+      - NIM_MODEL_PROFILE=vllm-fp16-tp2
+      - NIM_MANIFEST_ALLOW_UNSAFE=1
```

Comment:
Why are we specifying unsafe instead of relying on what is built into NIM? Do the GA profiles not include vllm-fp16-tp2 and not support the Vertex environment?

Reply:
Yes vllm-fp16-tp2 is compatible with GA and supported by VertexAI. Here specify NIM_MODEL_PROFILE variable to give user option to choose preferred compatible profile, or use a custom engine (e.g. TRT-LLM L4 custom built engine). Here the example use vLLM profile since TRTLLM on L4 instance has no current customer demands. 

NIM_MANIFEST_ALLOW_UNSAFE is an environment variable in previous NIM guide 'enable selection of a model profile not included in the original model_manifest.yaml or a profile that is not detected to be compatible with the deployed hardware.'. The usage here should be to use airgap_model_manifest.yaml.